### PR TITLE
[SPARK-8551][ML] Elastic net python code example

### PR DESCRIPTION
**Pull request by coderxiang**

### CODE REVIEW

**Comment by coderxiang** [comment]
jenkins, test this please

**Comment by coderxiang** [comment]
@jkbradley @dbtsai

File: examples/src/main/python/ml/logistic_regression.py [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+    stringIndexer = StringIndexer(inputCol="label", outputCol="indexedLabel")
+    si_model = stringIndexer.fit(df)
+    td = si_model.transform(df)
+    [training, test] = td.randomSplit([0.7, 0.3])
+
+    lr = LogisticRegression(maxIter=100, regParam=0.3).setLabelCol("indexedLabel")
+    lr.setElasticNetParam(0.8)
+
+    # Fit the model
+    lrModel = lr.fit(test)
```
Comment (jkbradley) [inline examples/src/main/python/ml/logistic_regression.py]: Fit on training, not test

**Comment by jkbradley** [comment]
Looks good other than that one comment

**Comment by dbtsai** [comment]
This is for LoR, how about LiR? Thanks.

**Comment by sujkh85** [comment]
NAVER - http://www.naver.com/
--------------------------------------------

sujkh@naver.com 님께 보내신 메일 <Re: [spark] [SPARK-8551][ML] Elastic net python code example (#6946)> 이 다음과 같은 이유로 전송 실패했습니다.

--------------------------------------------

받는 사람이 회원님의 메일을 수신차단 하였습니다. 


--------------------------------------------

**Comment by dbtsai** [comment]
LGTM

**Comment by coderxiang** [comment]
jenkins, test this please

**Comment by coderxiang** [comment]
@dbtsai yeah, I will do LiR with Java and python together!

**Comment by JoshRosen** [comment]
Jenkins, retest this please.

**Comment by dbtsai** [comment]
Merged into master. Thanks.

**Pull request closed without merging**